DevelopGitHub/GitHubExtension/GitHubExtension.Notes.WebApi/Controllers/NoteController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Threading.Tasks;
+using System.Web.Http;
+using GitHubExtension.Notes.WebApi.Services;
+using GitHubExtension.Notes.WebApi.ViewModels;
+using Microsoft.AspNet.Identity;
+
+namespace GitHubExtension.Notes.WebApi.Controllers
+{
+    public class NoteController : ApiController
+    {
+        private INotesService noteService;
+
+        public NoteController(INotesService noteService)
+        {
+            this.noteService = noteService;
+        }
+
+        [Route("api/note/{noteId}")]
```

Comment:
Will make it a constant, but wait for a merge where class for routes constants is added to develop